modules/docs/src/Volo.Docs.Domain/Volo/Docs/DocsDomainModule.cs

```diff
@@ -21,7 +22,8 @@ namespace Volo.Docs
     [DependsOn(
         typeof(DocsDomainSharedModule),
         typeof(AbpDddDomainModule),
-        typeof(AbpAutoMapperModule)
+        typeof(AbpAutoMapperModule),
+        typeof(AbpBackgroundJobsModule)
```

Comment:
We are adding `AbpBackgroundJobsModule` here, but adding AbpBackgroundJobsAbstractionModule to the application layer. `AbpBackgroundJobsModule` already depends on the abstraction package and the application depends on the domain. Please revisit the dependencies.